### PR TITLE
[RFR] Rename add command to setall, simplify syntax

### LINKED ISSUE #53: comfy add should set the stable configuration, not the next
In the general case, the developer should not have to use tags

### PR DESCRIPTION
**Pull request by fzaninotto**
image

Closes #53.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,11 +87,11 @@ Configuration saved locally in .comfy/config
 comfy project successfully created
 ```
 
-Add a configuration from a config file for a particular environment using `comfy add`:
+Set the configuration for a particular environment based on the content of an existing file using `comfy seyall`:
```
Comment (Kmaschta) [inline README.md]: s/seyall/setall/

File: cli/src/commands/setall.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+module.exports = (ui, modules) => function* (rawOptions) {
+    const { red, green, bold } = ui.colors;
+    const options = minimist(rawOptions);
+    const env = options._[0];
+    const configPath = options._[1];
+    const tag = options.tag || options.t || 'stable';
+
+    if (options.help || options.h || options._.includes('help')) {
+        help(ui);
+        return ui.exit(0);
```
Comment (Kmaschta) [inline cli/src/commands/setall.js]: No need of this line

Comment (fzaninotto) [inline cli/src/commands/setall.js]: In fact, the ui.exit should be removed from the help function

Comment (fzaninotto) [inline cli/src/commands/setall.js]: There, I removed it. That way, the control statements are all located in the same (main) function.

Comment (Kmaschta) [inline cli/src/commands/setall.js]: I did agree with you about the `--help` that should not execute the command!
But the `ui.exit` instruction after the `help(ui)` one was redundant.

Comment (fzaninotto) [inline cli/src/commands/setall.js]: Why? It's the main program's job to exit, which is what it does...

**Comment by fzaninotto** [comment]
Validated face to face with @Kmaschta

**Pull request merged**